AutomatticTracks/src/main/java/com/automattic/android/tracks/crashlogging/internal/SentryEventExt.kt

```diff
@@ -0,0 +1,9 @@
+package com.automattic.android.tracks.crashlogging.internal
+
+import io.sentry.SentryEvent
+
+internal fun SentryEvent.appendTags(tags: Map<String, String>) {
+    tags.forEach { (key, value) ->
+        this.setTag(key, value)
+    }
+}
```

Comment:
[`SentryEvent#setTags` will override all previous tags](https://github.com/getsentry/sentry-java/blob/4.3.0/sentry/src/main/java/io/sentry/SentryBaseEvent.java#L139). I've created this extension as we add tags of `application context` as well as we can add tags for a single event.